nucleus/pom.xml

```diff
@@ -46,31 +46,16 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>net.java</groupId>
-        <artifactId>jvnet-parent</artifactId>
-        <version>5</version>
-        <relativePath />
+        <groupId>fish.payara</groupId>
```

Comment:
Then I would have to update all childrens. And I even don't believe the "server" is useful here, it is even controversial, because "micro" is "serverless".

Reply:
Ok let's leave nucleus as is for now, we'll move it with next great renaming.